tests/common/include/tests/common/tools_cylinder_intersection.inl

```diff
@@ -55,6 +55,26 @@ TEST(ALGEBRA_PLUGIN, translated_cylinder) {
                 hit_bound.p2[1] != not_defined);
     ASSERT_NEAR(hit_bound.p2[0], 0., isclose);
     ASSERT_NEAR(hit_bound.p2[1], -5., isclose);
+    ASSERT_NEAR(hit_bound.cos_incidence_angle, 1., isclose);
+}
+
+// This defines the local frame test suite
+TEST(ALGEBRA_PLUGIN, cylinder_incidence_angle) {
+    // Create a translated cylinder and test untersection
+    const transform3 shifted(vector3{0., 0., 0.});
```

Comment:
With a translation of (0,0,0) this is exactly not a shifted cylinder, right?

Reply:
fixed